src/test/test-time-util.c

```diff
@@ -701,6 +701,16 @@ static void test_parse_timestamp_impl(const char *tz) {
         test_parse_timestamp_one("70-01-01 00:00:01.001 UTC", 0, USEC_PER_SEC + 1000);
         test_parse_timestamp_one("70-01-01 00:00:01.0010 UTC", 0, USEC_PER_SEC + 1000);
 
+        /* Examples from RFC3339, except the one from 1937 since it's before the epoch */
```

Comment:
please add all examples, and check that the one before the epoch results in a parse error, correctly.